### DISCUSSION TITLE
How do I share my project?

### DISCUSSION THREAD

**Discussion by chambersDon** [microsoft/BotFramework-Composer#9602] [Q&A]
I have a composer project built, but other developers need to work on it.

How do I share this project?
It's not deployed to Azure yet.

**Comment by ram-xv** [comment]
Hi @chambersDon,

Composer has a built-in feature that exports your bot assets to a .zip file.

Here are the steps:
1. Open your bot in Bot Framework Composer.
2. On the left directory explorer, click on the bot's `•••` (more options) button.
    ![image](https://github.com/microsoft/BotFramework-Composer/assets/38049078/89e1a8ab-def5-42d6-bd11-fc84162ad92b)
3. In the dropdown, select "Export assets to .zip".
4. This will create a self-contained ZIP file containing all of your declarative assets making up your Composer project.

After these steps, you will have a .zip file that contains all the necessary files for your bot. You can then share this .zip file with your coworkers, and they can import it into their own instance of Bot Framework Composer.
